app/config/variables.php

```diff
@@ -444,6 +444,15 @@
                 'question' => '',
                 'filter' => ''
             ],
+            [
+                'name' => '_APP_FUNCTIONS_OUTPUT_MAX',
+                'description' => 'The maximum length of characters a function execution is allowed to output. The default value is 65535',
+                'introduction' => '0.9.0',
+                'default' => '65535',
```

Comment:
This is the maximum anyway right?

Reply:
Yeah it is the max currently using the TEXT Type in MariaDB. Not sure about the rewrite though.

Reply:
Lets have this in the description - otherwise people might get confused when they raise this value 👍🏻

Reply:
Done :+1: